.github/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,23 @@
+
+## ✅ 작업 내용
+
+- [ ] 이런 일을
+- [ ] 했습니다
+
+## 🧪 테스트 방법
+
+- ex) console.log
+- ex2) .env
+
+## 📷 참고 자료 - 스크린샷 / 링크 / GIF / ...
```

markdownlint-cli2 (0.18.1)

Trailing punctuation in heading
Punctuation: ' ...'

(MD026, no-trailing-punctuation)